extensions/google-drive/src/utils.ts

```diff
@@ -42,6 +42,14 @@ export const getDriveRootPath = (): string => {
 
   return resolve(preferences.googleDriveRootPath.trim().replace("~", homedir()));
 };
+export const getExcludePaths = (): string[] => {
```

Comment:
```suggestion
export const getExcludePaths = (): Array<string> => {
```

I prefer this style. :)